### DISCUSSION TITLE
有没有办法能查看整合包是什么版本的

### DISCUSSION THREAD

**Discussion by bakamoon** [Meloong-Git/PCL#3912] [提问] [answered]
![QQ图片20240527125908](https://github.com/Hex-Dragon/PCL2/assets/89714943/05af6ba1-102f-4554-8ccc-2ae1093c858a)
像是这样，整合包后面跟着一个整合包版本号，想确认自己的整合包是哪个版本需不需要更新最新的

**Comment by LTCatt** [comment]
只能做到在安装后塞一点东西到版本介绍里，HMCL 这个应该是在安装过程中把版本号内置了

**Reply by LTCatt** [reply, marked as answer]
转 #3913
